app/views/components/button.scala.html

```diff
@@ -16,6 +16,6 @@
 
 @this(govukButton: GovukButton)
 
-@(msg: String)(implicit messages: Messages)
+@(msg: String, href: Option[String] = None)(implicit messages: Messages)
```

Comment:
Could you please write a test class for this as we are making change to the view structure?